web/config/sync/core.entity_form_display.node.article.default.yml

```diff
@@ -0,0 +1,68 @@
+uuid: 5104975f-3316-4924-b200-ba7e78a9fd9e
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.article.field_article_type
+    - field.field.node.article.field_author
+    - field.field.node.article.field_content
+    - node.type.article
+  module:
+    - paragraphs
+id: node.article.default
+targetEntityType: node
+bundle: article
+mode: default
+content:
+  created:
+    type: datetime_timestamp
+    weight: 5
+    settings: {  }
+    third_party_settings: {  }
+  field_article_type:
+    weight: 1
+    settings: {  }
+    third_party_settings: {  }
+    type: options_select
+  field_author:
+    weight: 2
+    settings:
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    type: string_textfield
+  field_content:
+    type: entity_reference_paragraphs
+    weight: 3
+    settings:
+      title: Content
+      title_plural: Content
+      edit_mode: preview
+      add_mode: button
+      form_display_mode: default
+    third_party_settings: {  }
+  promote:
+    type: boolean_checkbox
+    settings:
+      display_label: true
+    weight: 6
+    third_party_settings: {  }
+  sticky:
```

Comment:
Can we remove the sticky and promote fields? They do not seem to make much sense for us.